kitsune/sumo/forms.py

```diff
@@ -0,0 +1,51 @@
+from django import forms
+from django.conf import settings
+from timeout_decorator import TimeoutError
+from waffle import switch_is_active
+
+from kitsune.sumo.utils import is_toll_free_number, match_regex_with_timeout
+
+
+class KitsuneBaseForumForm(forms.Form):
+    """Base form suitable for all the project.
+
+    Mainly adds a common clean method to deal with spam.
+    """
+
+    def __init__(self, *args, **kwargs):
+        """Override init method to get the user if possible."""
+        self.user = kwargs.pop('user', None)
+        super(KitsuneBaseForumForm, self).__init__(*args, **kwargs)
+
+    def clean(self, *args, **kwargs):
+        """Generic clean method used by all forms in the question app.
+
+        Parse content for any toll free numbers.
+        """
+        cdata = self.cleaned_data.get('content')
+        if not cdata:
+            return super(KitsuneBaseForumForm, self).clean(*args, **kwargs)
+
+        # At the moment validation doesn't reach this point b/c the logout and deactivation
+        is_spam = is_toll_free_number(cdata)
+
+        if is_spam:
+            self.cleaned_data.update({
+                'is_spam': True
+            })
```

Comment:
Since this won't start working without removing the existing behaviour, should we just comment it out/remove it for now?